clientv3/balancer/grpc1.7-health.go

```diff
@@ -507,7 +507,7 @@ func (b *GRPC17Health) Get(ctx context.Context, opts grpc.BalancerGetOptions) (g
 		addr = b.pinAddr
 		b.mu.RUnlock()
 		if closed {
-			return grpc.Address{Addr: ""}, nil, grpc.ErrClientConnClosing
+			return grpc.Address{Addr: ""}, nil, context.Canceled
```

Comment:
why do we need to change this error type?

Reply:
@spzala 

```
use he status code of Canceled instead of grpc.ErrClientConnClosing which is deprecated
```

Is it deprecated in gRPC? We haven't bumped up the gRPC yet.

Reply:
@gyuho thanks and yes, the deprecated one. I also found a related issue https://github.com/etcd-io/etcd/issues/9304 so seems like we are good for now and I will make updates here. Thanks both!